articles/libraries/lock-android/v1/error-messages.md

```diff
@@ -0,0 +1,16 @@
+---
+description: Types of error messages with Lock for Android
+---
+# Android Lock: Error Messages
+
+<%= include('../_includes/_lock-version') %>
+
+You may encounter one or more of the following messages when signing up using Lock for Android:
+
+* user_exists: the user you are attempting to sign up has already signed up;
+* username_exists: the username you are attempting to sign up with is already in use;
+* invalid_user_password: the password you have chosen does not meet the requirements;
+* unauthorized: you may not sign up for this application;
+* a0.mfa_required
```

Comment:
mfa_required: MFA is required for this account. You must retry the log in sending the MFA code.